j2v/utils/helpers.py

```diff
@@ -33,22 +32,21 @@ def is_str_timestamp(dim_val):
     :return: True only if string represents a timestamp
     """
     try:
-        if datetime.datetime.strptime(dim_val, "%Y-%m-%dT%H:%M:%S.%fZ"):
+        if datetime.strptime(dim_val, "%Y-%m-%dT%H:%M:%S.%fZ").isoformat():
```

Comment:
Why the addition of `isoformat`?

Reply:
That was a requirement, to check for the support of ISO format of date and time.